lib/config/loadConfig.ts

```diff
@@ -8,9 +8,14 @@ function readConfigFile(configPath: string) {
   return {};
 }
 
-export function loadConfig(configPath: string): Config {
-  return {
-    ...defaultConfig,
-    ...readConfigFile(configPath)
-  };
+export async function loadConfig(configPath: string): Promise<object> {
```

Comment:
Wy not `Promise<Config>`?

Reply:
Originally, because the if the user option is empty, then return an empty object, now I will change to returning a `defaultConfig` and take your comment. thanks